requirements.txt

```diff
@@ -36,17 +38,22 @@ rdflib==4.2.1
 selenium==2.47.1
 coverage==3.7.1
 pyasn1==0.1.8
+pycparser==2.14
+cffi==1.2.1
```

Comment:
This causes a failure on snowball: `fatal error: ffi.h: No such file or directory`

Looks like we'll need to add the `libffi-dev` package to our django servers in salt to use this package.